submissions/natashafir/a-tiny-js-world/index.js

```diff
@@ -0,0 +1,49 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+function Inhabitants(species, name, gender, legs, hands, saying) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.legs = legs;
+    this.hands = hands;
+    this.saying = saying;
+    this.sayHi = function() {
+        print( "My name is " + this.name + ". I'm " + this.species + ". My gender is " + this.gender +
```

Comment:
In OOP classes and created objects must remain unaware of the environment. `print` is a function specific to the environment. Imagine we want to out inhabitant's self-presentation in multiple ways - web page, console, some file. What would you change so that the object would not care about the ways of delivering information to a consumer and yet consumer would be able to decide how to use inhabitant's self-presentation?